sklearn/utils/estimator_checks.py

```diff
@@ -4012,3 +4014,62 @@ def check_transformer_get_feature_names_out_pandas(name, transformer_orig):
     assert (
         len(feature_names_out_default) == n_features_out
     ), f"Expected {n_features_out} feature names, got {len(feature_names_out_default)}"
+
+
+def check_param_validation(name, estimator_orig):
+    # Check that an informative error is raised when the value of a constructor
+    # parameter does not have an appropriate type or value.
+    rng = np.random.RandomState(0)
```

Comment:
use ` np.random.RandomState(global_random_seed)` instead. Ref: https://github.com/scikit-learn/scikit-learn/issues/22827

Reply:
This fixture is not meant to be used on tests that checks for errors and basic properties that does not actually depend on the data.

Reply:
Also, pytest is not required for estimator_checks, so I don't think it would work anyway